src/fu-util.c

```diff
@@ -356,51 +361,60 @@ fu_util_perhaps_show_unreported(FuUtilPrivate *priv, GError **error)
 	/* show the success and failures */
 	if (!priv->assume_yes && !all_automatic) {
 		/* delimit */
-		g_print("________________________________________________\n");
+		fu_console_println_literal(priv->console,
+					   "________________________________________________");
 
 		/* failures */
 		if (devices_failed->len > 0) {
-			/* TRANSLATORS: a list of failed updates */
-			g_print("\n%s\n\n", _("Devices that were not updated correctly:"));
+			fu_console_println_literal(priv->console,
+						   /* TRANSLATORS: a list of failed updates */
+						   _("Devices that were not updated correctly:"));
 			for (guint i = 0; i < devices_failed->len; i++) {
 				FwupdDevice *dev = g_ptr_array_index(devices_failed, i);
 				FwupdRelease *rel = fwupd_device_get_release_default(dev);
-				g_print(" • %s (%s → %s)\n",
-					fwupd_device_get_name(dev),
-					fwupd_device_get_version(dev),
-					fwupd_release_get_version(rel));
+				fu_console_println(priv->console,
+						   " • %s (%s → %s)",
+						   fwupd_device_get_name(dev),
+						   fwupd_device_get_version(dev),
+						   fwupd_release_get_version(rel));
 			}
 		}
 
 		/* success */
 		if (devices_success->len > 0) {
-			/* TRANSLATORS: a list of successful updates */
-			g_print("\n%s\n\n", _("Devices that have been updated successfully:"));
+			fu_console_println_literal(
```

Comment:
@superm1 the other way to do this is have fu_console_print("%s\n", _("Devices that h..` but I don't know if that's better or worse.

Reply:
I like this the current way